contrib/examples/apiserver/broker.yaml

```diff
@@ -8,6 +8,7 @@ spec:
   # service-catalog will use the contents of the secret. The secret should have "username"
   # and "password" keys
   authInfo:
-    basicAuthSecret:
+    secretRef:
```

Comment:
Current approach also replicates the way `Secret.Type` is implemented: https://github.com/kubernetes/kubernetes/blob/master/pkg/api/types.go#L3841